src/StarcounterClientFiles/wwwroot/sys/app-shell/app-shell.html

```diff
@@ -3,13 +3,14 @@
 <head>
     <meta charset="utf-8">
     <title>{0}</title>
-
     <script src="/sys/webcomponentsjs/webcomponents-lite.js"></script>
-    <link rel="import" href="/sys/polymer/polymer.html">
-    <link rel="import" href="/sys/starcounter.html">
-    <link rel="import" href="/sys/starcounter-include/starcounter-include.html">
-    <link rel="stylesheet" href="/sys/underwear.css/underwear.css">    
+    <link rel="import" href="/sys/starcounter.min.html">
```

Comment:
This will break the middlewares in BlendingProvider:  

- https://github.com/Starcounter/Blending/blob/1baf4f8e1def5e929942cad223bbdb719c44fce7/src/BlendingEditor/Composition/Middleware.cs#L20
- https://github.com/Starcounter/Blending/blob/6daec49cfaf6e10bd21e7451bb71a629ae4f264d/src/BlendingProvider/Composition/Api/Middleware.cs#L11

Can we keep `/sys/starcounter.min.html`?